<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeCalendarContainer</class>
 <widget class="QWidget" name="RangeCalendarContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>284</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainBk" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalTop">
        <item>
         <widget class="QToolButton" name="btnLastDay">
          <property name="text">
           <string>近1天</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnLastWeek">
          <property name="text">
           <string>近1周</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnLastFifteenDays">
          <property name="text">
           <string>近15天</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnLastMonth">
          <property name="text">
           <string>近1个月</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnLastThreeMonth">
          <property name="text">
           <string>近3个月</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalTopMore"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnYes">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>40</number>
        </property>
        <item>
         <widget class="RangeCalendarWidget" name="calendarStart">
          <property name="verticalHeaderFormat">
           <enum>QCalendarWidget::NoVerticalHeader</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="RangeCalendarWidget" name="calendarEnd">
          <property name="verticalHeaderFormat">
           <enum>QCalendarWidget::NoVerticalHeader</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="tips">
          <property name="text">
           <string>提示：左边圆形为起始位，右边圆形为末位</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RangeCalendarWidget</class>
   <extends>QCalendarWidget</extends>
   <header>range_calendar_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
